src/app/(protected)/teacher/[teacherId]/components/nav-main.tsx

```
@@ -0,0 +1,50 @@
+}) => {
+  const { user } = useAppSelector((state) => state.user);
+
+  const pathname = usePathname();
+
+  return (
+    <SidebarMenu className="gap-2">
+      {items.map((item) => (
+        <SidebarMenuItem key={item.title}>
+          <SidebarMenuButton
+            asChild
+            isActive={pathname === item.href.replace(":teacherId", user?._id!)}
```

Biome (2.3.14)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)